Snowdrift.cabal

```diff
@@ -7,7 +7,7 @@ maintainer:        dev@lists.snowdrift.coop
 synopsis:          Infrastructure for Snowdrift fundrasing site.
 category:          Web
 stability:         Experimental
-cabal-version:     >= 1.8
+cabal-version:     >= 1.20
```

Comment:
per my MR, [GG-26](https://git.gnu.io/snowdrift/snowdrift/merge_requests/26), please add a short comment line saying:

    -- Cabal 1.20 is needed to recognize cabal.config files